_migrate-or-upgrade/rolling-upgrade/index.md

```diff
@@ -1,19 +1,23 @@
 ---
 layout: default
 title: Rolling upgrade
-parent: Upgrading OpenSearch
-nav_order: 10
+nav_order: 20
+has_toc: true
+permalink: /migrate-or-upgrade/rolling-upgrade/
+redirect_from: 
+ - /upgrade-opensearch/appendix
```

Comment:
_migrate-or-upgrade/rolling-upgrade/appendix/index.md has the same redirect. Please have each page redirect to only one page.